src/components/Helpjuice/Helpjuice.tsx

```diff
@@ -58,6 +58,15 @@ export const Helpjuice: React.FC = () => {
       link.setAttribute('target', '_blank');
       link.textContent = nameHeader.textContent;
       nameHeader.replaceChildren(link);
+
+      // Helpjuice renders links to questions as relative URLs, e.g. href="/question/123"
+      // We need to turn them into absolute URLs so that they open correctly
+      for (const link of document.querySelectorAll('#article-content-body a')) {
+        const href = link.getAttribute('href');
+        if (href?.startsWith('/')) {
+          link.setAttribute('href', `${helpJuiceOrigin}${href}`);
+        }
```

Comment:
You can make the browser's native CSS selector do the heavy lifting
```suggestion
      const relativeLinks = document.querySelectorAll('#article-content-body a[href^="/"]');
      for (const link of relativeLinks) {
        link.setAttribute('href', `${helpJuiceOrigin}${link.getAttribute('href')}`);
      }
```

Reply:
Good idea!